Applications/Internet/Internet Explorer 7.0/Online/script.js

```diff
@@ -20,7 +20,8 @@ var installerImplementation = {
             .version(LATEST_STABLE_VERSION)
             .distribution("upstream")
```

Comment:
`distribution()` and `architecture()` must be removed, they do not exist anymore